### ISSUE TITLE
Android: Printing

### ISSUE THREAD

**Issue by mark-prins** [openmsupply/openmsupply-client#1247]
This error pops up when printing on android:

![print-error](https://user-images.githubusercontent.com/9192912/168708748-6524edd2-338f-45e8-ba78-539045dc6aae.png)

**Comment by clemens-msupply** [comment]
yes good point, it tries the directory within the package which is read only. Would have to change it to the app file directory. We already know this directory so its "just" a matter of making the printing output path configurable and passing it on to the printing service.

**Comment by craigdrown** [comment]
I'm just thrilled the error wasn't "Chrome devtools doesn't support printing on Android" ;-)

**Comment by clemens-msupply** [comment]
> I'm just thrilled the error wasn't "Chrome devtools doesn't support printing on Android" ;-)

I fixed the path issues but then I get:
"could not autodetect a chrome executable"
I can't find any information that Android is supported for headless chrome...
